deploy/docker-compose-build.yml

```diff
@@ -21,9 +23,10 @@ services:
       - ballerine-workflow-service
     restart: on-failure
     environment:
-      VITE_API_URL: 'http://localhost:3000/api/v1/'
+      VITE_API_URL: 'http://${DOMAIN_NAME}:3000/api/v1/'
```

Comment:
if the domain name is undefined it should probably use just a path, in this case "/api/v1/"